src/mon/Monitor.h

```diff
@@ -635,8 +635,8 @@ class Monitor : public Dispatcher,
     return (class MgrStatMonitor*) paxos_service[PAXOS_MGRSTAT];
   }
 
-  class MgrStatMonitor *healthmon() {
-    return (class MgrStatMonitor*) paxos_service[PAXOS_MGRSTAT];
+  class HealthMonitor *healthmon() {
```

Comment:
is this method used anywhere?

Reply:
it wasn't before, but it is now in Monitor::win_election

Reply:
ahh, i see.